src/app/components/Toolbar/Features/Account/ImportableAccount.tsx

```diff
@@ -17,10 +17,6 @@ export const ImportableAccount = ({
       displayBalance={true}
       displayCheckbox={true}
       displayAccountNumber={true}
-      displayDerivation={{
```

Comment:
I kinda liked the derivation path. Any reason it is going away?

Reply:
With Don we were trying to improve illegible "BLE Ledger":
- we tried and liked an icon https://icons.grommet.io/?s=bluetooth instead
- we didn't find the best icon for usb https://icones.js.org/collection/all?s=usb but could also just keep "USB"
- balance position depends on the account type, and longer derivation paths needed fixing
  ![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/079aa4ee-ab65-46d3-93b7-d19249711546)

We also don't show it inside account selector, where it would actually add more information:
![image](https://github.com/oasisprotocol/oasis-wallet-web/assets/3758846/db02244e-ad96-4d8b-88e1-3f4e02caf5b8)

Removing it in import and keeping it in account manage and expanding BLE to Bluetooth was simplest. But I like derivation too :/